src/huggingface_hub/hf_api.py

```diff
@@ -9768,6 +9912,9 @@ def _parse_revision_from_pr_url(pr_url: str) -> str:
 list_spaces = api.list_spaces
 space_info = api.space_info
 
+list_papers = api.list_papers
+paper_info = api.paper_info
```

Comment:
These two must be added at the root of `huggingface_hub` package. To do so, you need to add them [to this list](https://github.com/huggingface/huggingface_hub/blob/1b8e9cf4b1f55be4d11370071f3b15e4c04ba915/src/huggingface_hub/__init__.py#L140) and run `make style` which will make sure alphabetical order is respected + add a type checking annotation. You can then commit the changes.